dasharo-compatibility/cpu-status.robot

```diff
@@ -23,7 +23,7 @@ Suite Setup         Run Keywords
 ...                     AND
 ...                     Power On    AND
 ...                     Enter Setup Menu Tianocore    AND
-...                     Reset To Defaults Tianocore    AND
+# ...               Reset To Defaults Tianocore    AND
```

Comment:
Rather not intended change?

Reply:
Yes. Commented only for testing purposes. Fixed: 0eb2859d045c806e608c7da819b8302de0e9775c